src/devices/Mcp25xxx/Register/BitTimeConfiguration/Cnf1.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp25xxx.Register.BitTimeConfiguration
+{
+    /// <summary>
+    /// Configuration 1 Register.
+    /// </summary>
+    public class Cnf1 : IRegister
+    {
+        /// <summary>
+        /// Initializes a new instance of the Cnf1 class.
+        /// </summary>
+        /// <param name="brp">Synchronization Jump Width Length bits.</param>
+        /// <param name="sjw">
+        /// Baud Rate Prescaler bits.
+        /// TQ = 2 x (BRP[5:0] + 1)/FOSC.
+        /// </param>
+        public Cnf1(byte brp, SynchronizationJumpWidthLength sjw)
+        {
+            if (brp > 0b0011_1111)
+            {
+                throw new ArgumentException($"Invalid BRP value {brp}.", nameof(brp));
+            }
+
+            Brp = brp;
+            Sjw = sjw;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the Cnf1 class.
+        /// </summary>
+        /// <param name="value">The value that represents the register contents.</param>
+        public Cnf1(byte value)
+        {
+            Brp = (byte)(value & 0b0011_1111);
+            Sjw = (SynchronizationJumpWidthLength)((value & 0b1100_0000) >> 6);
+        }
+
+        /// <summary>
+        /// Synchronization Jump Width Length.
+        /// </summary>
+        public enum SynchronizationJumpWidthLength
+        {
+            /// <summary>
+            /// Length = 1 x TQ.
+            /// </summary>
+            Tqx1 = 0,
+            /// <summary>
+            /// Length = 2 x TQ.
+            /// </summary>
+            Tqx2 = 1,
+            /// <summary>
+            /// Length = 3 x TQ.
+            /// </summary>
+            Tqx3 = 2,
+            /// <summary>
+            /// Length = 4 x TQ.
+            /// </summary>
+            Tqx4 = 3,
+        }
+
+        /// <summary>
+        /// Baud Rate Prescaler bits.
+        /// TQ = 2 x (BRP[5:0] + 1)/FOSC.
+        /// </summary>
+        public byte Brp { get; }
```

Comment:
Since you made these public I'd extend the names

Reply:
I will update this, but after my next group of commits.

Reply:
Updated